### DISCUSSION TITLE
从 Auto Layout 的布局算法谈性能 · /layout-performance

### DISCUSSION THREAD

**Discussion by draveness** [draveness/blog-comments#274] [Announcements]
https://draveness.me/layout-performance

**Comment by WeMadeCode** [comment]
深度好文👍👍👍，目前遇到了这个问题

**Comment by CanyeGuOO** [comment]
请问，ASDK flex布局列表不复用，那么如果列表有几万个甚至更多的cell， 内存会爆掉怎么处理？

**Comment by Luciennnnnnn** [comment]
感谢博主的好文。 不过不得不吐槽一下“所以其时间复杂度就是多项式时间的，不难推测出，在处理极其复杂的 UI 界面时，会造成性能上的巨大损失。”这句话。。 多项式就是最好的时间复杂度了，log也属于多项式。
